### ISSUE TITLE
[Bug] Archived pictures will be reuploaded when app is reinstalled

### ISSUE THREAD

**Issue by ffuuuuu** [immich-app/immich#11812]
### The bug

With the newest app I had an issue where images showed up with a blurred preview and no picture in the timeline. Clearing the cache didn't help, so I had to reinstall the app.
Found out when relinking the local picture folder all the pictures that were already archived on the server where reuploaded to the timeline when starting the sync process.
The bug would be that the pictures aren't marked as archived already.

### The OS that Immich Server is running on

Android

### Version of Immich Server

v.1.112.1

### Version of Immich Mobile App

v.1.122.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default compose
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1. Sync a media folder and archive some pictures on the mobile android app
2. Delete the app and reinstall it
3. Sync the folder again and only the archived pictures get reuploaded to the timeline. The others show with a checkmark.
...
```

**Comment by bo0tzz** [comment]
When reinstalling the app, this is currently expected behaviour. It'll hopefully be improved in the future.

**Comment by alextran1502** [comment]
Are you describing that you archived asset in Immich and they get reupload? This is archiving, not deleting, correct?

**Comment by ffuuuuu** [comment]
> Are you describing that you archived asset in Immich and they get reupload? This is archiving, not deleting, correct?

Well the pictures are already in the archive and get added again to the normal timeline. But sadly this is expected behavior and won't be fixed for a while.

**Comment by sgloutnikov** [comment]
I am subscribed to track potential fixes to this problem in #1553. Last I tried the iOS app  got the cloud check-mark in the photos (signaling it was uploaded), but still went on to re-upload the entire archive to the server. I have not reinstalled recently, but looks like that behavior still persists.
